### PR TITLE
Test Pull Request

### PR DESCRIPTION
**Pull request by PerenDarksbane**
"Sup"

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -13,3 +13,11 @@ jobs:
     - uses: actions/checkout@v2
     - name: Dummy command
       run: touch foo 
+#    - name: First one fails # this one does fail and causes the next run to be skipped!
```
Comment (plankp) [inline .github/workflows/test.yml:16]: Let's say this line is actually wrong or sth and I want to give feedback...?

**Review by plankp — CHANGES_REQUESTED** [review]
what about this?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -13,3 +13,11 @@ jobs:
...
+#      run: |
+#        false
+#        true
+#     - name: Seconnd one fails # this one also fails
```
Comment (plankp) [inline .github/workflows/test.yml:20]: Say, turns out this line is also wrong but I didnt see it during the first round of reviews...

**Review by plankp — COMMENTED** [review]
DANG IT!

**Comment by plankp** [comment]
Learned how the GitHub code review workflow is... Thanks~

(Closing this PR)

File: .github/workflows/test.yml (hunk 1 above)
Comment (plankp) [reply .github/workflows/test.yml:16]: and respond

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
```
Comment (plankp) [inline .gitignore:1]: And what does "Add single comment" do?

Comment (PerenDarksbane) [reply .gitignore:1]: Yup I can see this too

**Pull request closed without merging**